lib/category-name-converter.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
```

Comment:
@scottnath @conortm While things like 'use strict' placement might bite you in the ass, most coding standard rules won't. Their value really comes in preventing psycho ocd where a dev rewrites an entire file by their own personal standard.

A higher level discussion of what these code repos do should really happen sooner than later. However, we don't have qa reviewing any of this, so in their lieu, I am running the unit tests and trying to get a bead on what each function does. The higher level discussion may not be necessary as a pre-requisite for merging this pr.